mvlearn/ajive/ajive.py

```diff
@@ -0,0 +1,760 @@
+                                                   [bn]['individual'],
+                                                   noise=block_specific[bn]\
+                                                   ['noise'],
+                                                   block_name=bn,
+                                                   obs_names=obs_names,
+                                                   var_names=var_names[bn],
+                                                   m=self.centers_[bn],
+                                                   shape=blocks[bn].shape)
+
+        return self
+
+    @property
+    def is_fit(self):
+        if hasattr(self, "blocks"):
+            return True
+        else:
+            return False
+
+    @property
+    def block_names(self):
+        if self.is_fit:
+            return list(self.blocks.keys())
+        else:
+            return None
+
+    def save(self, fpath, compress=9):
+        dump(self, fpath, compress=compress)
+
+    @classmethod
+    def load(cls, fpath):
+        return load(fpath)
+
+    def get_full_block_estimates(self):
+        """
+
+        Output
+        ------
+        full: dict of dict of np.arrays
+        The joint, individual, and noise full estimates for each block.
+
+        """
+        full = {}
+        for bn in self.block_names:
+            full[bn] = {'joint': self.blocks[bn].joint.full_,
+                        'individual': self.blocks[bn].individual.full_,
+                        'noise': self.blocks[bn].noise_}
+
+        return full
+
+    def results_dict(self):
+        """
+        Returns all estimates as a dicts.
+
+        """
+        results = {}
+        results['common'] = {'scores': self.common.scores_,
+                             'svals': self.common.svals_,
+                             'loadings': self.common.loadings_,
+                             'rank': self.common.rank}
+
+        for bn in self.block_names:
+            joint = self.blocks[bn].joint
+            indiv = self.blocks[bn].individual
+
+            results[bn] = {'joint': {'scores': joint.scores_,
+                                     'svals': joint.svals_,
+                                     'loadings': joint.loadings_,
+                                     'rank': joint.rank,
+                                     'full': joint.full_},
+
+                           'individual': {'scores': indiv.scores_,
+                                          'svals': indiv.svals_,
+                                          'loadings': indiv.loadings_,
+                                          'rank': indiv.rank,
+                                          'full': indiv.full_},
+
+                           'noise': self.blocks[bn].noise_}
+
+        return results
+
+    def get_ranks(self):
+        """
+        Output
+        ------
+        joint_rank (int): the joint rank
+
+        indiv_ranks (dict): the individual ranks.
+        """
```

Comment:
These are not formatted correctly. Should be

joint_rank : int
    The joint rank.
indiv_ranks : dict
    The individual ranks.